src/components/posts/OrderPosts.js

```diff
@@ -21,46 +18,40 @@ function OrderPosts({
   return (
     <div className={classes.order}>
       <div className={classes.actions}>
-        {orderUpDown ? (
-          <button onClick={upHandler}>UP</button>
-        ) : (
-          <button onClick={upHandler} style={{ backgroundColor: "blue" }}>
-            UP
-          </button>
-        )}
-      </div>
-
-      {authorDate ? (
-        <>
-          <div>
-            <button onClick={dateHandler} style={{ backgroundColor: "blue" }}>
+        {authorDate ? (
+          <>
+            <button
+              onClick={dateHandler}
+              style={{ backgroundColor: "#ba7a92" }}
```

Comment:
This might be cleaner to move the ternary outside of the return, then pass in the style background colour to both and change that instead, means that the buttons and the structure would stay consistent and might make it look cleaner, still likely personal preference though